resources/gamesettings.json

```diff
@@ -65,6 +65,11 @@
             "Randomize relationship values when creating Clan",
             "Clan founder cats will start the game with established relationships.",
             true
+        ],
+        "they them default": [
+            "Use they/them as default pronouns",
+            "If this setting is on new cats will generate with they/them pronouns, regardless of gender.",
+            true
```

Comment:
The default for this setting should be `false`. This is a huge project -- it shouldn't be hidden behind a non-default toggle! :)